docs/data/xrootd-authorization.md

```diff
@@ -0,0 +1,141 @@
+Configuring XRootD Authorization
+================================
+
+There are several authorization options in XRootD available through the security plugins. 
+In this document, we will cover the security option supported in the OSG:
+
+-   [`xrootd-lcmaps`](#recommended-security-option-xrootd-lcmaps-authorization): Using the LCMAPS callout to use
+    X509-based authentication
+
+Note: On the data nodes, the files will actually be owned by unix user `xrootd` (or other daemon user), not as the user
+authenticated to, under most circumstances. 
+XRootD will verify the permissions and authorization based on the user that the security plugin authenticates you to,
+but, internally, the data node files will be owned by the `xrootd` user.
+
+#### Authorization file
+
+In order to add security to your cluster you will need to add "auth\_file" on the your data server node. 
+Create `/etc/xrootd/auth_file` :
+
+```file
+# This means that all the users have read access to the datasets 
+u * %RED%/data/xrootdfs%ENDCOLOR% lr
+
+# This means that all the users have full access to their private dirs 
+u = %RED%/data/xrootdfs/%ENDCOLOR%@=/ a
+
+# This means that this privileged user can do everything 
+# You need at least one user like that, in order to create the 
+# private dir for each user willing to store his data in the facility 
+u xrootd %RED%/data/xrootdfs%ENDCOLOR% a 
+```
+
+Here we assume that your storage path is `/data/xrootdfs` (same as in the
+previous example).
+
+Change file ownership (if you have created file as root):
+
+```console
+root@host # chown xrootd:xrootd /etc/xrootd/auth_file
+```
+
+
+This file is a flat file of the following form:
+
+``` file
+idtype id path privs
+```
+
+| Item   | Description                                                                                                                           |
```

Comment:
"Field" is probably a better descriptor than "item"

Reply:
Updated.